portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -27,6 +27,6 @@
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello world!</h1>");
+    response.getWriter().println("<h1>Hello Jason!</h1>");
```

Comment:
Let's remove the HTML tags here...since you're using `.innerText`, they won't render well. You can get them to render using `.innerHTML`, but that has its own set of problems (see https://medium.com/@jenlindner22/the-risk-of-innerhtml-3981253fe217). Also, from an architectural standpoint, it's better to just return raw data from the servlet and leave the formatting to the UI (you can accomplish the same effect by CSS styling the DOM element you're modifying).

Reply:
Oops, I jumped the gun. This comment really pertains more to the Step 2 assignment. But it's fine to change it now anyway.

Reply:
I don't see any .innerText in this file. Isn't .innerText/.innerHTML just for formatting elements, but here I'm just printing a text?

Reply:
sorry the comment wasn't clear. The `.innerHTML` occurs when you add it to the DOM (in script.js)

Reply:
I see. I changed .innerText to .innerHTML in the script.js file, so I guess I can keep the tags here?

Reply:
Yes, your use of innerHTML with an HTML response from the servlet is consistent. For the reasons I outlined in my first comment on this thread, I'd prefer you switch to pure text from the servlet + innerText rendering + CSS styling to achieve the same effect, but I'll go ahead and approve this PR in the interests of unblocking you. You can switch it over if you get bored :). Great job!